llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskSchedulerService.java

```diff
@@ -1716,7 +1716,7 @@ protected void registerRunningTask(TaskInfo taskInfo) {
         isGuaranteed ? guaranteedTasks : speculativeTasks;
     writeLock.lock();
     try {
-      WM_LOG.info("Registering " + taskInfo.attemptId + "; " + taskInfo.isGuaranteed);
+      WM_LOG.info("Registering " + taskInfo.attemptId + ", guaranteed: " + taskInfo.isGuaranteed);
```

Comment:
Can we replace concat with logger format, like:
```
      WM_LOG.info("Registering {}, guaranteed: {}", taskInfo.attemptId, taskInfo.isGuaranteed);
```

Reply:
Done.